api/cpp/cmake/SlintMacro.cmake

```diff
@@ -14,7 +36,8 @@ function(SLINT_TARGET_SOURCES target)
 
             add_custom_command(
                 OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${_SLINT_BASE_NAME}.h
-                COMMAND Slint::slint-compiler ${_SLINT_ABSOLUTE}
+                COMMAND ${CMAKE_COMMAND} -E env "SLINT_EMBED_RESOURCES=${_SLINT_EMBED_RESOURCES}"
```

Comment:
Another option would be to add a command line option to the compiler. But I have no strong preference on this - just a thought :)

Reply:
A command line option to the compiler would be the most clean, IMHO